recipes/intake-cmip/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "intake-cmip" %}
+{% set version = "2019.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8c01220a15841f6cd99795919b61bf22072598df9a7d343fd8b66399a772879d
+  
+
+build: 
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+  skip: True  # [py2k]
```

Comment:
We should build this as noarch python. Do the following. 

1. Remove the skip: True line.
2. Replace it with `noarch: python`